tests/unit/lms/authentication/__init___test.py

```diff
@@ -1,5 +1,4 @@
 import pytest
-from pyramid import testing
```

Comment:
I ran an import cleaner on the tests to get rid of some I knew I'd left in, but it picked up loads more besides.